ditto/cli.py

```diff
@@ -23,6 +23,39 @@ def cli(ctx, verbose):
     ctx.obj = {}
     ctx.obj['verbose'] = verbose
 
+@cli.command()
+@click.option("--input", type=click.Path(exists=True), help="Path to input file")
+@click.option("--from", help="Convert from OpenDSS, Cyme, Gridlab-D, Demo, JSON")
+@click.option("--output", type=click.Path(exists=True), help="Path to the metrics output file")
+@click.option("--to", help="Format for the metrics output file. xlsx or json")
+@click.option("--feeder", default=True, type=bool, help="If True computes metrics per feeder. Otherwise, compute metrics at the system level.")
+@click.pass_context
+def metric(ctx, **kwargs):
+    """Compute metrics"""
+
+    verbose = ctx.obj['verbose']
+
+    for entry_point in iter_entry_points("ditto.readers"):
```

Comment:
If we are using this `entry_point` pattern in multiple places, we should extract it to `utils.py`.